src/process.cr

```diff
@@ -4,6 +4,10 @@ require "c/sys/resource"
 require "c/unistd"
 
 class Process
+  # A platform-specific identifier of a running process.
+  # Int64 is enough to fit underlying values of different platforms (Int32 or UInt32).
+  alias PID = Int64
```

Comment:
I don't think this alias is really helpful. It suggests a distinct type safe type when it is really just an alias of `Int64`.
I'd rather make it explicit as `Int64` or use an actual value type `record PID, value : Int64`.

Reply:
My goal here is to make it easy to find usages of PID in source code, rather than digging through Int64s. I don't see why not.

Reply:
I don't like the false type-safety it suggests. You would assume these methods return and receive only instances of `PID`. But that doesn't exist, it's just `Int64`.

Reply:
OK. How do we answer the inevitable question "why does it return Int64 if pids are Int32"?

Reply:
Documentation shouldn't be an issue. It can just go into the class' doc comment.

Reply:
I'll wait for a tie-breaker comment :upside_down_face:

Reply:
I don't see a problem with this.

Reply:
Could you clarify which you prefer?

Reply:
I think using a `record PID` is best, if the API returns such things and accepts such things. Then it can have `to_i` and `to_i64` if needed, and we use it internally, and people can use that too if needed. And then if we need to change the underlying type it should be safe.

But maybe it's overkill, I don't know.

Reply:
I don't see a problem with it as it is now - `alias PID = Int64`. After all, we do the same with `Bytes`.

Reply:
But `Bytes` is conceptually just a slice of UInt8, so this really just a different name fore xactly the same thing. But for a PID type I would expect some specific properties, that are not shared by Int64. For example a narrower value range. And that I can't accidentally assign arbitrary int values, but would be required to explicitly cast a number to PID.

Reply:
Bikeshedding. There's no need to care deeply about this minor detail. This is good enough, let it be.

Reply:
I disagree on being a minor detail. To me it's essential for approving this PR. When PID is just Int64, the only benefit is improved discoverability in the API docs. But at the cost of suggesting a false type safety. I would probably prefer calling it Int64 directly then.